sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpEventHubClient.cs

```diff
@@ -398,23 +483,15 @@ private async Task<string> AquireAccessTokenAsync(CancellationToken cancellation
         }
 
         /// <summary>
-        ///   Determines if a given AMQP message response is an error and, if so, throws the
-        ///   appropriate corresponding exception type.
+        ///   Uses the minimum value of the two specified <see cref="TimeSpan" /> instances.
         /// </summary>
         ///
-        /// <param name="response">The AMQP response message to consider.</param>
-        /// <param name="eventHubName">The name of the Event Hub associated with the request.</param>
+        /// <param name="firstOption">The first option to consider.</param>
+        /// <param name="secondOption">The second option to consider.</param>
         ///
-        private static void ThrowIfErrorResponse(AmqpMessage response,
-                                                 string eventHubName)
-        {
-            var statusCode = default(int);
-
-            if ((response?.ApplicationProperties?.Map.TryGetValue(AmqpResponse.StatusCode, out statusCode) != true)
-                || (!AmqpResponse.IsSuccessStatus((AmqpResponseStatusCode)statusCode)))
-            {
-                throw AmqpError.CreateExceptionForResponse(response, eventHubName);
-            }
-        }
+        /// <returns></returns>
```

Comment:
Missing description.

Reply:
Caught that in the producer changes, will float it in with those.